preupdater_template.sh

```diff
@@ -45,6 +47,77 @@ device_repin() {
     curl --fail --retry 10 -s -X PATCH "${API_ENDPOINT}/v2/device(${DEVICE_ID})" -H "Authorization: Bearer ${DEVICE_API_KEY}" -H "Content-Type: application/json" --data-binary '{"build":'"${BUILD_ID}"'}' || finish_up "Couldn't set new commit hash."
 }
 
+#######################################
+# Globals:
+#   CONFIG_PATH
+# Arguments:
+#   var_name: the name of the required entry in config.json
+# Returns:
+#   var_value: the value of the entry
+#######################################
+resin_var_manual_load() {
+    local var_name=$1
+    local var_value
+
+    if [ -f "${CONFIG_PATH}" ]; then
+        var_value=$(jq -r ".${var_name}" "${CONFIG_PATH}") || finish_up "Couldn't get device API key."
+        if [ ! -n "${var_value}" ]; then
+            finish_up "Couldn't load device API key manually."
```

Comment:
nit: since this is used for several variables other than deviceApiKey, the error message could be something like `"Couldn't load ${var_name} from config.json"

Reply:
@pcarranzav yeah, sorry, this was during a refactor :P Will clean up!